scripts/release-checks/dependency-ranges/peer-deps-check.mts

```diff
@@ -0,0 +1,74 @@
+/**
+ * @license
+ * Copyright Google LLC All Rights Reserved.
+ *
+ * Use of this source code is governed by an MIT-style license that can be
+ * found in the LICENSE file at https://angular.io/license
+ */
+
+import path from 'path';
+import url from 'url';
+import semver from 'semver';
+import { PackageMap } from '../../../lib/packages.js';
+
+/** Path to the current directory. */
+const currentDir = path.dirname(url.fileURLToPath(import.meta.url));
+
+/** Path to the project directory. */
+const projectDir = path.join(currentDir, '../../../');
```

Comment:
Consider: Could we resolve the `package.json` file to find the project root instead of manually maintaining `../` paths? Would be easy to move this file and forget to update it. Not sure if there's an easy way to get the path of the current `package.json` without walking the directory tree, but something to consider.

Reply:
That's the common pattern we follow in other repos too. You would need to adjust relative imports as well, so not something new here 😆